client/components/happychat/style.scss

```diff
@@ -136,6 +145,83 @@
 
 
 /**
+ * Minimize animation
+ */
+
+.layout .happychat .happychat__container.is-open.minimizing {
+	box-shadow: none;
+	border-top: 1px solid lighten( $gray, 20% );
+	border-left: 1px solid lighten( $gray, 20% );
+	border-right: 1px solid lighten( $gray, 20% );
+	background: lighten( $gray, 30% );
+	animation: minimize .5s 1 forwards;
+
+	> .happychat__loading,
+	> .happychat__composer,
+	> .happychat__welcome,
+	> .happychat__conversation,
+	> .happychat__title .happychat__active-toolbar * {
+		visibility: hidden;
+	}
+
+	> .happychat__title {
+		transform: translateZ(0);
+		position: relative;
+		animation: disappear .1s 1 forwards;
+	}
+
+}
+
+@keyframes disappear {
```

Comment:
Wonder if you'd want to add these to the new `Animate` component we have: https://github.com/Automattic/wp-calypso/tree/master/client/components/animate